crates/nostr-relay-builder/src/local/inner.rs

```diff
@@ -549,6 +551,24 @@ impl InnerLocalRelay {
                 subscription_id,
                 filter,
             } => {
+                // Check the subscription id length
+                match self.max_subid_length {
+                    Some(max) if subscription_id.as_str().chars().count() > max => {
+                        return send_msg(
+                            ws_tx,
+                            RelayMessage::Closed {
+                                subscription_id,
+                                message: Cow::Owned(format!(
+                                    "{}: subscription ID exceeds max length {max}",
+                                    MachineReadablePrefix::RateLimited
```

Comment:
I guess here should be "blocked" or "error", instead of "rate limited"?

Reply:
Yeah, I just copied it from the close below. :)